### ISSUE TITLE
Antiviral assay data for Catalogue and ASAP compounds

### ISSUE THREAD

**Issue by HadiaAmahli** [StructuralGenomicsConsortium/CNP25-CHIKV-nsP3-Macrodomain#7]
On 8th Aug 2024, [25 catalogue compounds and 10 ASAP](https://github.com/user-attachments/files/17100879/CHIKV_Nsp3_UNC_Antiviral_Assay_AViDDUCL-0017_Plate_Map.xlsx) compounds shipped to our collaborator (Marcia Kay Sanders) at UNC for antiviral assay, the data received on 12th Sep 2024 showing that there is only one compound [RA-0188429](https://github.com/user-attachments/assets/af4288c4-9e5d-4b7e-8061-efba61f7dbe7) which is a toolbox-ASAP- compound has EC50= 13.56 µM, all other compounds did not show any activity equal or below 40 µM.
After a great and long discussion between our team at UCL and DLS team, we were not convinced with these data since based on the principle of the macrodomain (MD) role which is explained by the removal of ADP ribose hydrolase activity of the protein which can induce the immune response of the protein, this immune response can not be deployed in the cellular model since MD does not kill the virus.
Regarding other studies which have been found in the [literature]([Iterative computational design and crystallographic screening identifies potent inhibitors targeting the Nsp3 Macrodomain of SARS-CoV-2 (biorxiv.org)](https://www.biorxiv.org/content/10.1101/2022.06.27.497816v2.full.pdf), there is another methods that we can rely one to calculate the IC50 accurately, this method is: Homogeneous Time Resolved Fluorescence assay assessed by the displacement of an ADPr. conjugated biotin peptide from His6-tagged protein using a HTRF-technology based screening assay. I suggested this method in our target meeting which has been held today 23rd Sep 2024 however it seems that it’s unavailable.
Our protein scientist at UCL @Wenjie-Ch  is setting up the Isothermal Titration Calorimetry experiment to be able to determine to determine the dissociation constant, KD.
We also can rely on the GCI and TSA data in addition to [the crystallography data]([CHIKV_Mac (diamond.ac.uk)](https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Ffragalysis.diamond.ac.uk%2Fviewer%2Freact%2Fpreview%2Ftarget%2FCHIKV_Mac%2Ftas%2Flb32633-6&data=05%7C02%7Ch.almahli%40ucl.ac.uk%7Cf63e6dde262b45ad0c7908dcd7fa230b%7C1faf88fea9984c5b93c9210a11d9a5c2%7C0%7C0%7C638622716585333394%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=TploYk4RfdXEXwGGGSo2VTG5FXQgzsLCmFPaB2Afn5k%3D&reserved=0) to select a good number of new analogues aiming to increase their MW and to move from Fragment-like compounds to high molecular weight compounds to improve their binding affinity with the protein as well as their IC50 values.

**Comment by jasminaschenbrenner** [comment]
Hi @HadiaAmahli, @rahmanszsaleem, 
can you please reiterate the reasons that it was decided to prioritise RA-0188454-02? Comparing it to the other two hits, I remember that RA-0188454-02 was more 'risky' as it forms mainly water bridges with the protein instead of direct interactions, so it sits not directly in the pocket, and it actually does not bind at all in a way it was predicted to, based on the fragments it originated from. 
Just so that we can follow the thought process.

**Comment by mattodd** [comment]
@jasminaschenbrenner yes indeed, but the orthogonal GCI assay showed a better curve than the other two, sadly. The other two compounds are also to be pursued, and can be accessed using the same chemistry.

(@HadiaAmahli can you add pictures to the actual issue (rather than as links) just by dragging them onto the Issue? That way we can see everything together.

The slide I presented at the f2f meeting is this onem below. Does this still accurately reflect the state of play with respect to there being boxes around the compounds that gave good GCI curves?

<img width="1068" alt="Screenshot 2024-10-15 at 22 18 31" src="https://github.com/user-attachments/assets/d136770a-3cf6-4576-9e31-8686cfccdbd9">

Talking with Wenjie today, I think that compound Z9101663889, which gave a Jasmin Top Rating, has not yet been tested by GCI. We will need to fix that ASAP.

**Comment by HadiaAmahli** [comment]
@mattodd I arranged this table as an answer to your question above and which explains the status of the compounds highlighted in green.
![image](https://github.com/user-attachments/assets/a2e5cce3-3c06-42ea-986a-263bd3fe13d2)

Due to the old samples consumption, [new batch of the compounds has been purchased](https://github.com/user-attachments/files/17393318/Re-purchase.pures.for.Wenjie-26092024.xlsx) to enable @Wenjie-Ch  to run GCI experiment and to repeat the experiment for some compounds that showed confusing data comparing with the crystallography data: 
e.i: Z86430593 which is the top rated compound based on the crystal structure data, **it showed no binding on the GCI data**! 
![image](https://github.com/user-attachments/assets/fae41cc4-1bd7-4773-aa41-29d63e858499)
